### PR TITLE
Feature/dpp 90

### PR DESCRIPTION
**Pull request by sksadjad**
added the ssi-sdk.core function for `enablePostgresUuidExtension` and exported it. also made data-store use it

### CODE REVIEW

File: packages/data-store/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "typeorm-postgres:migration:run": "pnpm run typeorm -- migration:run -c migration-postgres"
   },
   "dependencies": {
+    "@sphereon/ssi-sdk.core": "workspace:*",
```
Comment (nklomp) [inline packages/data-store/package.json:16]: Pleases also update the references in tsconfig.json

**Review by nklomp — CHANGES_REQUESTED** [review]
Please update tsconfig

File: packages/ssi-sdk-core/src/utils/database.ts [hunk 2]
```diff
@@ -1,3 +1,20 @@
 export const flattenArray = <T>(args: { items: Array<T | Array<T>> }): Array<T> => args.items.flat() as Array<T>
 
 export const flattenMigrations = <T>(args: { migrations: Array<T | Array<T>> }): Array<T> => args.migrations.flat() as Array<T>
+
+/**
+ * It should accept queryRunner from the typeorm
+ */
+export const enablePostgresUuidExtension = async (queryRunner: any) => {
+  if (!queryRunner.query) {
```
[the remark this answers is not in the document]
Comment (sksadjad) [reply packages/ssi-sdk-core/src/utils/database.ts]: modified this to have a type which has query function

File: packages/data-store/package.json (hunk 1 above)
Comment (sksadjad) [reply packages/data-store/package.json:16]: added core to tsconfig of data-store

File: packages/ssi-sdk-core/src/utils/database.ts [hunk 3]
```diff
@@ -2,19 +2,18 @@ export const flattenArray = <T>(args: { items: Array<T | Array<T>> }): Array<T>
 
 export const flattenMigrations = <T>(args: { migrations: Array<T | Array<T>> }): Array<T> => args.migrations.flat() as Array<T>
 
-/**
- * It should accept queryRunner from the typeorm
```
Comment (nklomp) [inline packages/ssi-sdk-core/src/utils/database.ts]: Okay an now add the remark back to the type please, so people know you are creating a type that mimics TypeOrm

**Review by nklomp — APPROVED** [review]

**Pull request merged**